src/content_scripts/contentlib.js

```diff
@@ -35,7 +35,7 @@ export async function waitForElement(
  * Add listener to recrawl page on important changes
  */
 export function addPageChangeListener(listener) {
-  browser.runtime.onMessage.addListener(message => {
+  browser.runtime.onMessage.addListener(async message => {
```

Comment:
Did this change for some reason?

Reply:
Can you clarify your question? The change you're commenting on is a bugfix where the browser expects a promise to be returned and might otherwise display an error, easiest way to ensure a promise is just making it async.

Reply:
Ah ok. Just wondered if it might've been accidental.